arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md

```diff
@@ -0,0 +1,111 @@
+
+Each DAC member gets their own set of BLS public and private keys. Every member needs to create their own new and secure BLS keys. They should do this independently and ensure these keys are random and only for their use. If you need help generating BLS keys, please refer to our guide on generating keys in the Arbitrum documentation: [Generating BLS Keys](/run-arbitrum-node/data-availability-committees/deploy-das#step-1-generate-the-bls-keypair).
+
+The main blockchain (parent chain) needs to know all DAC members' names and public keys to validate the integrity of the data being batched and posted. A 'keyset' is a list of all DAC members' public keys. It also shows how many signatures are needed to approve a <a data-quicklook-from="data-availability-certificate">Data Availability Certificate</a>. This design lets the chain owner modify the DAC's membership over time and lets DAC members change their keys if needed. See [Inside AnyTrust](/how-arbitrum-works/inside-arbitrum-nitro#inside-anytrust) for more information.
+
+In the following section, we will provide a detailed guide on generating a Keyset corresponding to your individual set of keys and instructions for its subsequent configuration within the chain.
+
+### Batch poster configuration
+
+AnyTrust works with a group of Data Availability Servers, forming a committee that ensures transaction data is accessible. When setting up the Nitro <a data-quicklook-from="batch">Batch</a> Poster, you need to provide specific information for each committee member. This includes each member's URL, BLS public key, and a parameter known as `assumed-honest`. As mentioned, assumed-honest refers to the minimum number of committee members we trust.
+
+To ensure data is stored properly, a certain number of committee members need to confirm they have the data. This number is calculated as `K = (N + 1) - H`, where `N` is the total number of committee members and `H` is the minimum number of members assumed to be honest.
+
+Someone setting up an AnyTrust node would need to first set up the committee of Data Availability Servers, including generating their BLS keys. You can learn more about this topic [in this guide explaining the process of generating BLS keys ](run-arbitrum-node/data-availability-committees/deploy-das#step-1-generate-the-bls-keypair).
+
+Here is a sample of the `JSON` configuration, taken from a past configuration of <a data-quicklook-from="arbitrum-nova">Arbitrum Nova</a>, which was used with the batch poster configuration. Note that due to a quirk in a configuration library we use in Nitro, the `backends` field is an escaped JSON string with `url`, and `pubkey`. `pubkey` is the base64 encoded BLS public key of the committee member.
```

Comment:
backends is no longer an escaped json string and is just normal json now and can be formatted nicely https://github.com/OffchainLabs/nitro/pull/2427